lib/libutil/getlocalbase.c

```diff
@@ -38,8 +38,8 @@ __FBSDID("$FreeBSD$");
 #include <unistd.h>
 
 #ifndef _PATH_LOCALBASE
-#ifdef COMPAT_64BIT
-#define	_PATH_LOCALBASE	"/usr/local64"
+#ifdef COMPAT_libcompat
```

Comment:
Same comment as for LOCALBASE in include/paths.h.

Reply:
Also, I wonder why this doesn't use paths.h upstream rather than duplicating the logic.

Reply:
Oh, probably for bootstrapping on older FreeBSD; the paths.h define and this code came in around the same time

Reply:
Humm, that seems odd indeed.

Reply:
Except it's not a bootstrap library... so who knows